File: Mobile/MyApplication/app/src/main/java/com/example/ooar/discountproject/model/NotificationBranch.java
package com.example.ooar.discountproject.model;

/**
 * Created by Onur Kuru on 5.3.2016.
 */
public class NotificationBranch {

    private Integer id;
    private Notification notificationId;
    private Branch branchId;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Notification getNotificationId() {
        return notificationId;
    }

    public void setNotificationId(Notification notificationId) {
        this.notificationId = notificationId;
    }

    public Branch getBranchId() {
        return branchId;
    }

    public void setBranchId(Branch branchId) {
        this.branchId = branchId;
    }
}

File: Web/src/model/NotificationBranch.js
var Notification = require("../model/Notification");
var Branch = require("../model/Branch");

var NotificationBranch = project.db.define('NotificationBranch', {
  id: {type: 'serial'}
});

NotificationBranch.hasOne('Notification', Notification, {field: 'notificationId'});
NotificationBranch.hasOne('Branch', Branch, {field: 'branchId'});

module.exports = NotificationBranch;